src/dynamo.js

```diff
@@ -46,4 +48,28 @@ export class Dynamo {
     let params = { RequestItems: { OnlineUsers: userDeleteRequests } }
     await client.batchWriteItem(params).catch((e) => console.error('Error while deleting stale users', e))
   }
-}
\ No newline at end of file
+
+  async createGame(participants) {
+    const id = crypto.randomUUID()
+    const params = {
+      TableName: tableGames,
+      Item: marshall({
+        id: id,
+        participants: participants,
+        playerTurn: participants[Math.floor(Math.random())],
+        state: 'FINISHED'
+      })
+    }
+    await client.putItem(params)
+  }
+
+  async getStartedGames(userId) {
+    const params = {
+      TableName: tableGames
+    }
+    const games = await client.scan(params).catch((e) => console.log(e))
+    const regularGames = games.Items.map((game) => unmarshall(game))
+    const userGames = regularGames.filter((game) => game.participants.includes(userId))
+    return userGames.filter((game) => game.state === 'STARTED')
```

Comment:
This works, and we can leave it like this for now.

The issue with doing things this way is that we can easily fetch millions of items on our server and crash it. The way we avoid this is by using `FilterExpression`s [click](https://dynobase.dev/dynamodb-filterexpression/). 
Please make a backlog item for converting  all `dynamo.js` functions that employ server filtering, to database filtering